test/langtools/jdk/javadoc/doclet/TestGlobalHtml/pkg1/C2.java

```diff
@@ -0,0 +1,156 @@
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package pkg1;
+
+import java.io.IOException;
+
+/**
+ * <div inert>
+ *   <p>This content is inert and not interactable.</p>
+ *   <a href="https://openjdk.org/" accesskey="O" title="OpenJDK's Website" tabindex="0">
+ *     Visit OpenJDK's Website
+ *   </a>
+ * </div>
+ *
+ * <div>
+ *   <p>This content is interactable.</p>
+ *   <a href="https://openjdk.org/" accesskey="O" title="OpenJDK's Website" tabindex="0">
+ *     Visit OpenJDK's Website
+ *   </a>
+ * </div>
+ *
+ * A class comment for testing.
+ *
+ * <p lang="en" accesskey="D" autocapitalize="none" draggable="true" spellcheck="false">
+ * This class extends C1 and provides additional functionalities.
+ * </p>
+ *
+ * <table class="plain">
+ * <caption>Examples of methods in C2</caption>
+ * <thead>
+ *    <tr>
+ *       <th scope="col">Method Name
+ *       <th scope="col">Description
+ * </thead>
+ * <tbody>
+ *    <tr>
+ *       <td>performAction
+ *       <td>Performs a specified action.
+ *    <tr>
+ *       <td>calculateResult
+ *       <td>Calculates a result based on input.
+ * </tbody>
+ * </table>
+ *
+ * @since JDK 8
+ */
+public class C2 extends C1 {
+
+    /**
+     * Constructor with title and test flag.
+     *
+     * <p itemprop="constructor" itemtype="http://schema.org/Person">
+     * Initializes a new instance of the C2 class with the specified title and test flag.
+     * </p>
+     *
+     * @param title the title
+     * @param test  boolean value
+     */
+    public C2(String title, boolean test) {
+        super(title, test);
+    }
+
+    /**
+     * Constructor with title only.
+     *
+     * <p lang="en"> Initializes a new instance of the C2 class with the specified title.</p>
+     *
+     * @param title the title
+     */
+    public C2(String title) {
+        super(title);
+    }
+
+    /**
+     * Perform a specified action.
+     *
+     * <p contenteditable="true" draggable="true" spellcheck="true" data-method="action">
+     * This method performs an action and returns a status.
+     * </p>
+     *
+     * <ul>
```

Comment:
Since you use `ol`, `ul`, and `li`elements, you could add global attributes to them too. Otherwise, you only have such attributes in `p` elements.